sound/soc/sof/intel/hda-ipc.c

```diff
@@ -160,11 +161,10 @@ irqreturn_t hda_dsp_ipc4_irq_thread(int irq, void *context)
 			spin_unlock_irq(&sdev->ipc_lock);
 		} else {
 			/* Notification received */
-			struct sof_ipc4_msg data = {{ 0 }};
 
 			data.primary = primary;
 			data.extension = extension;
-			sdev->ipc->msg.rx_data = &data;
+			sdev->ipc->msg.rx_data = &null_data;
```

Comment:
@plbossart This is not a null data. We set data.primary and extension above. And it will lead to build error

Reply:
and this is valid for the previous change as well at line 79